### PR TITLE
Nitrous.io Quickstart button

### PR DESCRIPTION
**Pull request by yeouchien**
Added Nitrous Quickstart files so anyone can quickly spin up a free, cloud dev environment to start using this repo. Clicking the Quickstart button will automatically clone this repo and will install and configure any required packages.

### CODE REVIEW

**Comment by dsander** [comment]
Nice, that works really well. I noticed that the specs are not passing because `phantomjs` is not installed. Could include it in the post create script as well as adding the new files do our `.dockerignore`?

**Comment by yeouchien** [comment]
Sure. I've added `phantomjs` installation in post create script and added new files in `.dockerignore`.

File: nitrous-post-create.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+export LC_ALL=en_US.UTF-8
+export DEBIAN_FRONTEND=noninteractive
+
+sudo apt-get update
+sudo -E apt-get -q -y install mysql-server
+sudo apt-get -y install libmysqlclient-dev
+sudo apt-get clean
+
+npm install -g phantomjs
+cp .env.example .env
```
Comment (cantino) [inline nitrous-post-create.sh]: Doesn't the user need to set `APP_SECRET_TOKEN` and some of the `DATABASE_*` variables in `.env`?

**Comment by cantino** [comment]
Great idea @yeouchien!

**Comment by dsander** [comment]
@yeouchien Does the updated script work for you? I tried it two timed and it seems to time out during the installation of `phantomjs`. 
As I understand it the `README.nitrous.md` is only used in the IDE, would it be possible to create that file in the post-create script and not add it to the repository?

File: nitrous-post-create.sh (hunk 1 above)
Comment (yeouchien) [inline nitrous-post-create.sh]: I've added `pwgen` to generate `DATABASE_PASSWORD` and `APP_SECRET_TOKEN` in `.env`  and also use `huginn` as `DATABASE_USERNAME`.

**Comment by yeouchien** [comment]
@dsander Weird.. It worked for me yesterday but got time out today. Anyway I've updated the script to use `phantomjs-prebuilt` since it's recommended to do so (https://www.npmjs.com/package/phantomjs).

For `README.nitrous.md`, I will add it into `post-create` script later today.

**Comment by yeouchien** [comment]
@dsander Updated `nitrous-post-create.sh` script with `README.nitrous.md` in it.

**Comment by dsander** [comment]
Nice, tried it a couple of times and its working well now. Thanks @yeouchien!

**Pull request merged**